src/viking.js

```diff
@@ -1,11 +1,92 @@
 // Soldier
-class Soldier {}
+class Soldier {
+  constructor(health, strength) {
+    this.health = health;
+    this.strength = strength;
+  }
+  attack() {
+    return this.strength;
+  }
+  receiveDamage(damage) {
+    this.health -= damage;
+  }
+}
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+  constructor(name, health, strength) {
+    super();
```

Comment:
En este caso como tenemos que utilizar la herencia, la manera correcta sería `super(health, strength)` y dejar solo el `this.name = name`